### PR TITLE
UI and Footer

### PR DESCRIPTION
**Pull request by Ruixin126**
change the UI of PMV - EN, PMV- ASHRAE, Adaptive - ASHRAE, Adaptive - EN on pc platform, and footer in both mobile. pc platform
image
image
image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,6 +21,4 @@ RUN pip install --no-cache-dir -r requirements.txt
 EXPOSE 8080
 
 ENV DEBUG_DASH False
-ENV PORT_APP 8080
-
-CMD python app.py
```
Comment (FedericoTartarini) [inline Dockerfile:26]: why did you delete this command?

File: Pipfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-dash = ">=2.17.1,
+dash = "*"
```
Comment (FedericoTartarini) [inline Pipfile:7]: please remove this code `dash = *` since it is better to specify a version

File: Pipfile [hunk 3]
```diff
@@ -13,16 +13,14 @@ numpy = "*"
 psychrolib = "*"
 pythermalcomfort = "*"
 icecream = "*"
-pydantic = ">=2.8.2
-plotly = ">=5.23.0"
-scipy = ">=1.14.0
+pydantic = "*"
```
Comment (FedericoTartarini) [inline Pipfile:16]: your codebase was not up to date, you should pull the remove changes before doing a pull request since someone else has created a pull request before you.

File: Pipfile [hunk 4]
```diff
@@ -13,16 +13,14 @@ numpy = "*"
...
 
 [dev-packages]
 pytest-playwright = "*"
 pytest-xdist = "*"
 bump-my-version = "*"
 black = "*"
 
-
```
Comment (FedericoTartarini) [inline Pipfile:26]: do not commit empyt lines.

**Review by FedericoTartarini — CHANGES_REQUESTED** [review]
Thank you for working on this, however, you should first update your codebase before doing a pull request since someone else has committed their code using a pull request. It is also not clear to me why you are proposing to change the whole app.py file. You should only be modifying the part of the files related to your change and not the whole file. In its current stage it is very hard for me to review this pull request. Please pull the latest code base and then implement your changes from there. I also left a few comments in the code.

File: Dockerfile (hunk 1 above)
Comment (Ruixin126) [reply Dockerfile:26]: sorry for that, this may be a mistake due to merging code

File: Pipfile (hunk 4 above)
Comment (Ruixin126) [reply Pipfile:26]: This is because I duplicated the code of some files by deleting them as a result

File: Pipfile (hunk 3 above)
Comment (Ruixin126) [reply Pipfile:16]: Do I need close the pull request and pull the remove changes then pull the request?
